test/test_nn.py

```diff
@@ -15916,6 +15916,46 @@ def helper(n, c, h, w, kernel_size, stride=None):
         helper(10, 512, 31, 31, 3, stride=2)
         helper(1, 129, 8, 8, 3, stride=2)
 
+    @onlyNativeDeviceTypes
+    @dtypes(torch.half, torch.float, torch.double)
+    @onlyCUDA
+    def test_max_pool3d_ndhwc(self, device, dtype):
+        def helper(n, c, h, w, d, kernel_size, stride=None):
```

Comment:
can you also add some tests where `kernel_size` is not the same in all dimensions?